src/main/java/Main.java

```diff
@@ -158,6 +178,23 @@ public static void main(String... args) {
 
         registerShutdownHook(graphDb);
 
+        if (enableNeo4jWebServer) {
+            // Start Neo4jWebserver
+            // Deprecated but useful for debugging and exploring the database
+            LOGGER.info("Starting Neo4j WebServer on 127.0.0.1 Port: " + configNeo4jWebPort);
+            WrappingNeoServerBootstrapper neoServerBootstrapper;
+            GraphDatabaseAPI api = (GraphDatabaseAPI) graphDb;
+
+            ServerConfigurator config = new ServerConfigurator(api);
+            config.configuration()
+                    .addProperty(Configurator.WEBSERVER_ADDRESS_PROPERTY_KEY, "127.0.0.1");
```

Comment:
Are your sure we always want to run it on 127.0.0.1? This limits connections to the server, right?